src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -85,14 +103,20 @@ public Task StopAsync(CancellationToken cancellationToken)
         }
 
         // Called by base.Run when the service is ready to start.
+        /// <summary>
+        /// Raises the Start event when the service is ready to start.
+        /// </summary>
+        /// <param name="args">The command line arguments passed to the service.</param>
```

Comment:
I'm not sure if this `args` here is command line argument passed to service.

@eerhardt what do you think?

Reply:
Can we just use `/// <inheritdoc />` here instead?